doc/build/distributing.md

```diff
@@ -157,16 +157,6 @@ set `USE_SYSTEM_BLAS=1` and `USE_SYSTEM_LAPACK=1`, you should also set
 `LIBLAPACK=-l$(YOURBLAS)` and `LIBLAPACKNAME=lib$(YOURBLAS)`. Else, the
 reference LAPACK will be used and performance will typically be much lower.
 
-Compilation scripts
-===================
-
-The [julia-nightly-packaging](https://github.com/staticfloat/julia-nightly-packaging)
-repository contains multiple example scripts to ease the creation of
-binary packages. It also includes miscellaneous tools to do things such as
-fetching the last good commit that passed the
-[Travis](https://travis-ci.org/JuliaLang/julia/builds) tests.
-
-
```

Comment:
The above is completely outdated as far as I can tell. A lot of other things in this file seem to be outdated. E.g. it links to and discusses https://github.com/JuliaCI/PackageEvaluator.jl which has been archived, apparently in favor of a successor PkgEval.jl. Perhaps updating this is just a matter for search&replace, but since I know nothing about this tool, I wouldn't dare to do this. But perhaps someone who knows about the release / distribution process should go over this file and see what else is outdated?